src/components/elements/carousel/carousel.js

```diff
@@ -135,7 +159,7 @@ export const Carousel = ({
                         onClick={scrollPrev}
                         enabled={prevBtnEnabled}
                         style={chevron_left}
-                        is_reviews={chevron_color === 'black'}
+                        is_reviews={chevron_color === 'black' || is_displayed_on_mobile}
```

Comment:
```suggestion
                        is_reviews={is_displayed_on_mobile}
```

It's better to use this flag from now on. can you add this flag to carousel with black chevron 🙏🏻

Reply:
will do ~